Source/init.cpp

```diff
@@ -210,14 +210,15 @@ HWND init_find_mom_parent()
 
 void init_await_mom_parent_exit()
 {
-	DWORD v0; // edi
+	DWORD tick;

```

Comment:
```suggestion
	DWORD dwTicks;
```
like every where else

Reply:
Fuck it, it seams really dependant on what file you look at.

Reply:
xD